docs/docs_style_guide.md

```diff
@@ -0,0 +1,479 @@
+multiplication sign (−×) not the letter x.
+
+Write out and hyphenate fractions: two-thirds, three-quarters
+
+Write out decimal fractions as numerals. Use the same number format
+for a sequence: 0.75 and 0.45
+
+### Measurements
+
+Use SI units (not Imperial units).
+
+Use numerals and spell out measurements at first mention.
+
+Don't use a space between the numeral and abbreviated measurement:
+'3,500kg' not '3,500 kg'.
+
+Abbreviating kilograms to kg is fine.
+
+If the measurement is more than one word, eg 'kilometres per hour',
+then spell it out the first time it is used with the abbreviation.
+From then on, abbreviate. If it is only mentioned once, don't
+abbreviate.
+
+Use Celcius for temperature, eg 37℃.
+
+### Metaphors
+
+See 'words to avoid'.
+
+### Millions
+
+Always use million in money (and billion), eg '£138 million'. Use
+millions in phrases, eg 'millions of people'.
+
+However, don't use '£0.xx million' for amounts less than £1 million.
+
+### Money
+
+Use the £ and $ symbols: '£75', '$90'.
+
+Don't use decimals unless pence are included. For example, use
+'£75.50' but not '$75.00'.
+
+Write out 'pence' and 'cents' in full eg 'the user is charged 4 cents
+per MB of transfer'.
+
+Currencies are lower case.
+
+### Months
+
+See 'Dates'.
+
+## N
+
+### Numbers
+
+Use 'one' unless you're talking about a step, a point in a list, or another situation where using the numeral makes more sense. For example, like:
+
+> in point 1 of the design instructions'
+
+or this:
+
+> You'll see 5 examples of good LiveCode design practice
+>
+> There will be:
+>
+> * 4 examples from mobile app development
+> * 1 example from desktop app development
+
+Write all other numbers in numerals (including 2 to 9) except where
+it's part of a common expression and it would look strange, eg 'one or
+two of them'. Use common sense!
+
+If a number starts a sentence, write it out in full ('Thirty-four
+widgets are included in this package') except where it starts a title
+or subheading.
+
+For numerals over 999, insert a comma for clarity. 'It was over 9,000'.
+
+Spell out common fractions, such as one-half.
+
+Use a % sign for percentages, eg 50%.
+
+Use a 0 where there's no digit before the decimal point.
+
+Use '500 to 900' and not '500–900' (except in tables).
+
+Use MB not KB for anything over 1MK, eg 4MB not 4096KM. For under
+1MB, use KB, eg 569KB not 0.55MB.
+
+Keep it as accurate as possible, up to 2 decimal places. For example:
+'4.03MB'.
+
+#### Ordinal numbers
+
+Spell out first to ninth. After that, use 10th etc.
+
+In tables, use numerals throughout.
+
+## O
+
+### online
+
+One word.
+
+### online services
+
+Lower case unless part of a proper noun.
+
+## P
+
+### Per cent
+
+Use per cent not percent. Percentage is 1 word. Always use % with a
+number.
+
+### plain English
+
+All content in LiveCode documentation should be written in plain
+English. You should also make sure you use language your audience
+will understand.
+
```

Comment:
State that, for consistency, American spellings are preferred? And that as the audience is international, to avoid colloquialisms.